.gitignore

```diff
@@ -1 +1,3 @@
 /bazel-*
+.idea/*
```

Comment:
This is a merge from master. Please look at the exact file from master branch: I am not going to change any of the code merged. This will cause more conflicts later when we merge from / to master again. All these suggestion should be fixed only in master.

https://github.com/istio/proxy/blob/master/.gitignore